### PR TITLE
spellcheck changes

### PR DESCRIPTION
**Pull request by akanagar**

### CODE REVIEW

File: documentation/User-Stories.asciidoc [hunk 1]
```diff
@@ -53,10 +53,10 @@ As an invited guest I would like to receive an email - after somebody as invited
 
 === US: revoke accepted invite
 
-As an invited guest I would like to revoke my previous answer in order to inform the system and the inviter about my no showup
+As an invited guest I would like to revoke my previous answer in order to inform the system and the `inviter` about my no showup
 
 ==== AC:
- . the inviter and myself receives an email about my cancellation
+ . the `inviter` and myself receives an email about my cancellation
```
Comment (ssarmokadam) [inline documentation/User-Stories.asciidoc]: inviter -> dict it is valid word

File: documentation/User-Stories.asciidoc [hunk 2]
```diff
@@ -76,7 +76,7 @@ As a waiter I would like to search by reference number or email address for the
 
 ===== AC:
 . After entering the email the systems shows the number of the table. In case no reservation found, a message is shown.
-. Entered email address could be email of inviter or any invited guest.
+. Entered email address could be email of `inviter` or any invited guest.
```
Comment (ssarmokadam) [inline documentation/User-Stories.asciidoc]: same as above

File: documentation/User-Stories.asciidoc [hunk 3]
```diff
@@ -98,7 +98,7 @@ As a guest I want to filter the menu so that I only see the dishes I am interest
 ===== AC:
 . the guest can filter by
  .. type: starter | main dish | dessert; XOR; if nothing is selected all are shown (default value)
- .. veggy (yes|no|does not matter (default))
+ .. `veggy` (yes|no|does not matter (default))
```
Comment (ssarmokadam) [inline documentation/User-Stories.asciidoc]: veggy -> dict

File: documentation/User-Stories.asciidoc [hunk 4]
```diff
@@ -200,9 +200,9 @@ AC:
...
- . for each reservation the inviters email, the guests-emails, the number of accepts and declines, calculated table number, the reservation-id, reservation date-time and creation-datetime are shown
- . the default filter for all lists is the todays date for reservation datetime. this filter can be deleted.
- . only reservations and orders with reservation date in the future shall be available in this view. All other orders and reservation shall not be deleted; for data analytics those orders and reservation shall still exist in the system.
+ . for each reservation the `inviters` email, the guests-emails, the number of accepts and declines, calculated table number, the reservation-id, reservation date-time and creation-datetime are shown
+ . the default filter for all lists is the today's date for reservation datetime. this filter can be deleted.
+ . only reservations and orders with reservation date in the future shall be available in this view. All other orders and reservation shall not be deleted; for `data analytics` those orders and reservation shall still exist in the system.
```
Comment (ssarmokadam) [inline documentation/User-Stories.asciidoc]: is data analytics detected as spell check?

File: documentation/agile.asciidoc [hunk 5]
```diff
@@ -24,9 +24,9 @@ The Sprint Review meetings are time boxed to one hour for the four week Sprint.
 
 For this project the team aligned on not having a specific Sprint Retrospective meeting. The team is going to have a retrospective of a finished Sprint during the Sprint Review.
 
-=== Daily Standups
+=== `Daily Standups`
 
-The team aligned on having two weekly Standup meetings instead of a Daily Standup meeting. In comparison with the time boxed length of 15mins described in the CAF for this project the team extended the Standup meeting to 30mins. The content of the meetings remains the same.
+The team aligned on having two weekly `Standup` meetings instead of a Daily `Standup` meeting. In comparison with the time boxed length of `15mins` described in the CAF for this project the team extended the `Standup` meeting to `30mins`. The content of the meetings remains the same.
```
Comment (ssarmokadam) [inline documentation/agile.asciidoc]: 15mins -> try 15 mins
Standup-> Stand Up
30mins -> 30 mins

File: documentation/angular-ci.asciidoc [hunk 6]
```diff
@@ -99,40 +99,40 @@ In order to easily reuse the pipeline in other angular projects, all variables h
...
+    `repositoryId` = 'pl-nexus'
     // Nexus internal URL
-    repositoryUrl = 'http://nexus3-core:8081/nexus3/repository/maven-snapshots'
+    `repositoryUrl` = 'http://nexus3-core:8081/nexus3/repository/maven-snapshots'
     // Maven global settings configuration ID
-    globalSettingsId = 'MavenSettings'
+    `globalSettingsId` = 'MavenSettings'
     // Maven tool id
-    mavenInstallation = 'Maven3'
+    `mavenInstallation` = 'Maven3'
```
Comment (ssarmokadam) [inline documentation/angular-ci.asciidoc]: ideally source should not be added to backticks ... so block between

[source]
----
// code here
----
should be ignored

File: documentation/angular-ci.asciidoc [hunk 7]
```diff
@@ -154,9 +154,9 @@ This will be used as follows:
 +
 [source, groovy]
 ----
-sh """yarn ${buildScript}"""
+`sh """yarn ${buildScript}"""`
```
Comment (ssarmokadam) [inline documentation/angular-ci.asciidoc]: no need of backticks here... this should be ignored by regex from .spellcheck.yml

File: documentation/angular-ci.asciidoc [hunk 8]
```diff
@@ -178,7 +178,7 @@ This will be used as follows:
 +
 [source, groovy]
 ----
-sh """yarn ${lintScript}"""
+`sh """yarn ${lintScript}"""`
```
Comment (ssarmokadam) [inline documentation/angular-ci.asciidoc]: same as above

File: documentation/angular-design.asciidoc [hunk 9]
```diff
@@ -75,8 +75,8 @@ Dialog which opens as a result of fulfilling the booking form, it displays all t
 ===== Invitation-dialog
 Dialog which opens as a result of fulfilling the invitation form, it displays all the data of the booking with friends attempt, if everything is correct, the user can send the information or cancel if something is wrong.
 
-==== UserArea
-Group of dialogs with the proposal of giving some functionalities to the user, as login, register, change password or connect with Twitter.
+==== `UserArea`
+Group of `dialogs` with the proposal of giving some functionalities to the user, as login, register, change password or connect with Twitter.
```
Comment (ssarmokadam) [inline documentation/angular-design.asciidoc]: dialogs is in dict remove backticks

File: documentation/angular-design.asciidoc [hunk 10]
```diff
@@ -90,25 +90,25 @@ Dialog designed specifically to connect your user account with Twitter.
...
-==== ReservationCockpitComponent
+==== `ReservationCockpitComponent`
 Data table with all the reservations and a filter to search them, to show more info about that table you can click on it and open a dialog.
 
 ===== Reservation-dialog
 Complete display of data regarding the selected table and its guests.
 
 === Email Management
-As the application send emails to both guests and hosts, we choose an approach based on URL’s where the email contain a button with an URL to a service in the app and a token, front-end read that token and depending on the URL, will redirect to one service or another. For example: 
+As the application send emails to both guests and hosts, we choose an approach based on `URL’s` where the email contain a button with an URL to a service in the app and a token, front-end read that token and depending on the URL, will redirect to one service or another. For example:
```
Comment (ssarmokadam) [inline documentation/angular-design.asciidoc]: URL should not be in backtick

File: documentation/angular-testing.asciidoc [hunk 11]
```diff
@@ -58,17 +58,17 @@ describe('MenuComponent', () => {
...
 
-At this moment this is the case for most of the components, in the future, some work would be applied on this matter to have a full testing experience in MyThaiStar components.
+At this moment this is the case for most of the components, in the future, some work would be applied on this matter to have a full testing experience in `MyThaiStar` components.
 
 === Dialog components
 
-Dialog components are in a special category because they can not be tested normally. In the way Material implements the opening of dialogs, you have to create a component that will load into a dialog, to tell the module to load this components when needed, they have to be added into a special array category: _EntryComponents_. So, to test them, we need to import them in the test file as well.
+Dialog components are in a special category because they can not be tested normally. In the way Material implements the opening of `dialogs`, you have to create a component that will load into a dialog, to tell the module to load this components when needed, they have to be added into a special array category: `_EntryComponents_`. So, to test them, we need to import them in the test file as well.
```
Comment (ssarmokadam) [inline documentation/angular-testing.asciidoc]: dialogs to dict

File: documentation/java-ci.asciidoc [hunk 12]
```diff
@@ -102,17 +102,17 @@ In order to easily reuse the pipeline in other java projects, all variables have
 ----
 environment {
     // Directory with java project
-    javaDir = 'java/mtsj'
+    `javaDir` = 'java/mtsj'
```
Comment (ssarmokadam) [inline documentation/java-ci.asciidoc]: sourcecode should not be included in backticks

File: documentation/java-design.asciidoc [hunk 13]
```diff
@@ -354,14 +354,14 @@ Finally, as default users to start using the _My Thai Star_ app we are going to
 
 [source, java]
 ----
-auth.inMemoryAuthentication().withUser("waiter").password("waiter").roles("Waiter").and().withUser("user0").password("password").roles("Customer");
+`auth.inMemoryAuthentication().withUser("waiter").password("waiter").roles("Waiter").and().withUser("user0").password("password").roles("Customer");`
```
Comment (ssarmokadam) [inline documentation/java-design.asciidoc]: source code should not be in backticks

File: documentation/java-design.asciidoc [hunk 14]
```diff
@@ -418,11 +418,11 @@ We need to secure three services, that only should be accessible for users with
 
 As part of the token we are providing the user _Role_. So, when validating the token, we can obtain that same information and build a `UsernamePasswordAuthenticationToken` with username and the roles as collection of _Granted Authorities_.
 
-Doing so, afterwards, in the implementation class of the _logic_ layer we can set up the related methods with the _java security_ '@RolesAllowed' annotation to block the access to the resource to users that does not match the expected roles.
+Doing so, afterwards, in the implementation class of the _logic_ layer we can set up the related methods with the _java security_ `'@RolesAllowed'` annotation to block the access to the resource to users that does not match the expected roles.
 
 [source,java]
 ----
-@RolesAllowed(Roles.WAITER)
+`@RolesAllowed(Roles.WAITER)`
```
Comment (ssarmokadam) [inline documentation/java-design.asciidoc:425]: same as above

File: documentation/mrchecker.asciidoc [hunk 15]
```diff
@@ -10,7 +10,7 @@ MrChecker is a testing framework included in devonfw with several useful modules
 
 First of all download the repository.
 
-You must run My Thai Star frontend and backend application and modify your url to the front in `mrchecker/endtoend-test/src/resources/settings.properties`
+You must run My Thai Star `frontend` and backend application and modify your url to the front in `mrchecker/endtoend-test/src/resources/settings.properties`
```
Comment (ssarmokadam) [inline documentation/mrchecker.asciidoc]: frontend-> front-end or front end

File: documentation/mrchecker.asciidoc [hunk 16]
```diff
@@ -19,15 +19,13 @@ To run the e2e test you have two options:
 *The first option is using the command line in devonfw distribution*
 ```
 cd mrchecker/endtoend-test/
-mvn test -Dtest=MyThaiStarTest -Dbrowser=Chrome
-
+`mvn test -Dtest=MyThaiStarTest -Dbrowser=Chrome`
```
Comment (ssarmokadam) [inline documentation/mrchecker.asciidoc]: source in between ``` do not require to be added in single backtick

File: documentation/mrchecker.asciidoc [hunk 17]
```diff
@@ -19,15 +19,13 @@ To run the e2e test you have two options:
...
 ```
 optionally you can use it with a headless version or using another navigator:
 ```
 // chrome headless (without visual component)
-mvn test -Dtest=MyThaiStarTest -Dbrowser=ChromeHeadless
-// use firefox navigator
-mvn test -Dtest=MyThaiStarTest -Dbrowser=FireFox
+`mvn test -Dtest=MyThaiStarTest -Dbrowser=ChromeHeadless`// use firefox navigator
```
Comment (ssarmokadam) [inline documentation/mrchecker.asciidoc]: same as above

File: documentation/my-thai-star-publish.asciidoc [hunk 18]
```diff
@@ -16,18 +16,18 @@ However, only a folder is relevant to this topic: *MTS*
...
+*Note*: Have a look at this wiki over `https://github.com/devonfw/devon-ci/wiki/guide-devonci-jenkins-pipeline[here]` to get a basic idea on how to write pipeline scripts.
 
 . Checking out my-thai-star form GitHub
 +
 This stage will check out the source code directly from GitHub:
 +
 [source, groovy]
 ----
-git credentialsId:'github-devonfw-ci', url:'https://github.com/devonfw/my-thai-star/'
+git credentialsId:`'github-devonfw-ci'`, url:'https://github.com/devonfw/my-thai-star/'
```
Comment (ssarmokadam) [inline documentation/my-thai-star-publish.asciidoc]: no backticks for source

File: documentation/my-thai-star-publish.asciidoc [hunk 19]
```diff
@@ -73,7 +73,7 @@ or
 +
 [source, groovy]
 ----
-yarn test:ci
+`yarn test:ci`
```
Comment (ssarmokadam) [inline documentation/my-thai-star-publish.asciidoc]: same as above

File: documentation/my-thai-star-publish.asciidoc [hunk 20]
```diff
@@ -130,7 +130,7 @@ The complete Groovy script:
 node {
     stage 'Checking out my-thai-star from GitHub'
         node {
-            git branch: 'develop', credentialsId: 'github-devonfw-ci', url: 'https://github.com/devonfw/my-thai-star/'
+            git branch: 'develop', credentialsId: `'github-devonfw-ci'`, url: 'https://github.com/devonfw/my-thai-star/'
```
Comment (ssarmokadam) [inline documentation/my-thai-star-publish.asciidoc]: same as above

File: documentation/nodejs-design.asciidoc [hunk 21]
```diff
@@ -168,11 +168,11 @@ To configure the _login_ we will create an instance of _Authentication_ in the a
 app.post('/mythaistar/login', auth.auth);
 ----
 
-To verify the presence of the _Authorization token_ in the headers, we will register in the express the _Authentication.registerAuthentication_ middleware. This middleware will check if the token is correct, if so, it will place the user in the request and continue to process it. If the token is not correct it will continue processing the request normally.
+To verify the presence of the _Authorization token_ in the headers, we will register in the express the `_Authentication.registerAuthentication_` middleware. This middleware will check if the token is correct, if so, it will place the user in the request and continue to process it. If the token is not correct it will continue processing the request normally.
 
 [source, javascript]
 ----
-app.use(auth.registerAuthentication);
+`app.use(auth.registerAuthentication);`
```
Comment (ssarmokadam) [inline documentation/nodejs-design.asciidoc]: same as above

File: documentation/nodejs-design.asciidoc [hunk 22]
```diff
@@ -201,11 +201,11 @@ Following the https://jwt.io/introduction/[official documentation] the implement
 
 ==== Current User request
 
-To provide to the client with the current user data our application should expose a service to return the user details. In this case the _Authentication_ has a method called _getCurrentUser_ which will return the user data. We only need register it at express.
+To provide to the client with the current user data our application should expose a service to return the user details. In this case the _Authentication_ has a method called `_getCurrentUser_` which will return the user data. We only need register it at express.
 
 [source, javascript]
 ----
-app.get('/mythaistar/services/rest/security/v1/currentuser', auth.getCurrentUser);
+`app.get('/mythaistar/services/rest/security/v1/currentuser', auth.getCurrentUser);`
```
Comment (ssarmokadam) [inline documentation/nodejs-design.asciidoc]: no backtick for source code

File: documentation/nodejs-design.asciidoc [hunk 23]
```diff
@@ -218,11 +218,11 @@ We need to secure three services, that only should be accessible for users with
 
 - (POST) `/mythaistar/services/rest/ordermanagement/v1/order/filter`.
 
-To ensure this, the _Authorization_ class has the _securizedEndpoint_ method that guarantees access based on the role. This method can be used as middleware in secure services. As the role is included in the token, once validated we will have this information in the request and the middleware can guarantee access or return a 403 error.
+To ensure this, the _Authorization_ class has the `_securizedEndpoint_` method that guarantees access based on the role. This method can be used as middleware in secure services. As the role is included in the token, once validated we will have this information in the request and the middleware can guarantee access or return a 403 error.
 
 [source, javascript]
 ----
-app.use('/mythaistar/services/rest/ordermanagement/v1/order/filter', auth.securizedEndpoint('WAITER'));
+`app.use('/mythaistar/services/rest/ordermanagement/v1/order/filter', auth.securizedEndpoint('WAITER'));
```
Comment (ssarmokadam) [inline documentation/nodejs-design.asciidoc]: same as above

File: documentation/sap-hana-guide.asciidoc [hunk 24]
```diff
@@ -50,14 +50,14 @@ SELECT DATABASE_NAME,  ACTIVE_STATUS FROM SYS.M_DATABASES ORDER BY 1;
 Run the below for enabling the script server
 [source, sql]
 ----
-ALTER DATABASE HXE ADD 'scriptserver'
+`ALTER DATABASE HXE ADD 'scriptserver'`
```
Comment (ssarmokadam) [inline documentation/sap-hana-guide.asciidoc]: same as above

File: documentation/traefik-reverse-proxy.asciidoc [hunk 25]
```diff
@@ -12,10 +12,10 @@ Example of labels:
 [source,yaml]
 ----
 labels:
-    - "traefik.http.routers.angular.rule=PathPrefix(`/`)"
+    `- "traefik.http.routers.angular.rule=PathPrefix(`/`)"
```
Comment (ssarmokadam) [inline documentation/traefik-reverse-proxy.asciidoc]: same as above

File: documentation/traefik-reverse-proxy.asciidoc [hunk 26]
```diff
@@ -24,10 +24,10 @@ If you want to build the images from code, change to My Thai Star root folder an
 
 [source,bash]
 ----
-$ docker-compose -f docker-compose.traefik.yml up -d --build
+`$ docker-compose -f docker-compose.traefik.yml up -d --build`
```
Comment (ssarmokadam) [inline documentation/traefik-reverse-proxy.asciidoc]: same as above

**Review by ssarmokadam — CHANGES_REQUESTED** [review]

**Comment by ssarmokadam** [comment]
@akanagar  where is your updated .spellcheck.yml file?

File: documentation/User-Stories.asciidoc (hunk 1 above)
Comment (akanagar) [reply documentation/User-Stories.asciidoc]: @ssarmokadam  Suggested changes implemented. Please check.

**Pull request merged**